fedmsg_meta_fedora_infrastructure/pagure.py

```diff
@@ -284,10 +284,27 @@ def subtitle(self, msg, **config):
             return tmpl.format(user=user, project=project, fields=fields)
         elif 'pagure.project.user.added' in msg['topic']:
             new_user = msg['msg']['new_user']
+            access = msg['msg'].get('access', None)
```

Comment:
I am aware of that :p . I have seen it being used a lot in pagure, so i thought it might be helpful for readability. I will change it if you want.